test_calculator.py

```
@@ -0,0 +1,32 @@
+import unittest
+from calculator import Calculator
+
+class TestCalculator(unittest.TestCase):
+    def setUp(self):
+        self.calc = Calculator()
+
+    def test_calculate_square(self):
+        self.assertEqual(self.calc.calculate_square(4), 16)
+        self.assertEqual(self.calc.calculate_square(-2), 4)
+
+    def test_calculate_cube(self):
+        self.assertEqual(self.calc.calculate_cube(3), 27)
+        self.assertEqual(self.calc.calculate_cube(-2), -8)
+
+    def test_calculate_square_root(self):
```

[WARNING] The test_calculate_square method is missing a case where the input is zero. It should raise an error according to the function's logic.